tests/accordion-test.ts

```diff
@@ -0,0 +1,423 @@
+                        </div>
+                        <div class="acc-body">${item.body}</div>
+                    </div>
+                `
+            )}
+        </aalam-accordion>
+    `;
+
+    return accordionTemplate;
+};
+
+describe("AalamAccordion", () => {
+    it("initially opens items with the acc-active class", async () => {
+        const el = await litFixture(
+            createAccordionTemplate({ activecls: "acc-active" }, [
+                {
+                    title: "What is Artificial Intelligence?",
+                    body: "body 1",
+                    active: true,
+                },
+                {
+                    title: "What is Machine Learning?",
+                    body: "body 2",
+                    active: true,
+                },
+                { title: "title 3?", body: "body 3", active: false },
+                { title: "title 4?", body: "body 4", active: false },
+                { title: "title 5?", body: "body 5", active: false },
+            ])
+        );
+
+        expect(el.querySelectorAll(".container.acc-active").length).to.equal(2);
+
+        const activeItems = el.querySelectorAll(".acc-active");
+        activeItems.forEach((item) => {
+            expect(
+                getComputedStyle(item.querySelector(".acc-body")).display
+            ).to.equal("block");
+        });
+
+        const inactiveItems = el.querySelectorAll(
+            ".container:not(.acc-active) .acc-body"
+        );
+        inactiveItems.forEach((item) => {
+            expect(getComputedStyle(item).display).to.equal("none");
+        });
+    });
+
+    it("allows multiple items to be open if nomultiple is true then set as false", async () => {
+        const el = await litFixture(
+            createAccordionTemplate(
+                { activecls: "acc-active", nomultiple: true },
+                [
+                    {
+                        title: "What is Artificial Intelligence?",
+                        body: "body 1",
+                        active: true,
+                    },
+                    {
+                        title: "What is Machine Learning?",
+                        body: "body 2",
+                        active: false,
+                    },
+                    { title: "title 3?", body: "body 3", active: true },
+                    { title: "title 4?", body: "body 4", active: true },
+                    { title: "title 5?", body: "body 5", active: false },
+                ]
+            )
+        );
+        const titles = el.querySelectorAll(".acc-title");
+
+        expect(el.hasAttribute("nomultiple")).to.be.true;
+        expect(el.querySelectorAll(".acc-active").length).to.equal(1);
+        const active = el.querySelectorAll(".acc-active");
+        active.forEach((item) => {
+            expect(
+                getComputedStyle(item.querySelector(".acc-body")).display
+            ).to.equal("block");
+        });
```

Comment:
You had initially set active for two elements, better to check the rest of the body elements display is set to none.